hri/packages/nlp/nlp/assets/prompts.py

```diff
@@ -0,0 +1,114 @@
+from datetime import datetime
+
+import pytz
+
+CURRENT_CONTEXT = """
+Today is {CURRENT_DATE}.
+Your name is FRIDA (Friendly robotic interactive domestic assistant), a domestic assistant developed by RoBorregos.
+RoBorregos is the representative Robotic team from Tec de Monterrey, Campus Monterrey. It has around 40 members.
+You compete in the Robocup@home competition. Last summer you competed in the Netherlands, at the international competition. Last March you competed in TMR, obtaining 2nd place in Mexico.
+"""
+
+SYSTEM_PROMPT_CI_V2 = """
```

Comment:
this seems like a nice place to start with the xml prompts, not for now but just so I dont forget lol